mail_forward/views/compose_message.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- © 2014-2015 Grupo ESOC <www.grupoesoc.es>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record id="compose_form" model="ir.ui.view">
+    <field name="name">Forward Email Form</field>
+    <field name="model">mail_forward.compose_message</field>
+    <field name="inherit_id" ref="mail.email_compose_message_wizard_form"/>
+    <field name="arch" type="xml">
+        <data>
+            <!-- It's required to fill either `partner_ids` or
+                 `destination_object_id` -->
+            <xpath expr="//field[@name='partner_ids']" position="attributes">
```

Comment:
I always love standards. [XPath is one, backed by W3C](http://www.w3.org/TR/xpath/).

I also love using the right tool for the job. Quoting from that link before:

>  The primary purpose of XPath is to address parts of an XML document.

Which means that probably any developer that has needed to select a XML element in any app (not only Odoo) has met already XPath, which makes them less newbies in Odoo. I did not need to read any docs when I first saw this tag in an Odoo view, its purpose is obvious, but I needed to do it when using the shortcuts you say.

Any Odoo programmer will have to learn XPath sooner or later, since QWeb inheritance works only with it.

But the most important thing is:

![hakuna-matata](https://cloud.githubusercontent.com/assets/973709/10453020/60d07ebc-71a9-11e5-8f47-b69d0f426dd9.png)